lib/Runtime/Base/FunctionBody.cpp

```diff
@@ -6591,11 +6591,11 @@ namespace Js
         }
 
         // Simple JIT counts down and transitions on overflow
-        const uint8 callCount = simpleJitEntryPointInfo->callsCount;
+        const uint32 callCount = simpleJitEntryPointInfo->callsCount;
         Assert(simpleJitLimit == 0 ? callCount == 0 : simpleJitLimit > callCount);
         return callCount == 0 ?
             static_cast<uint16>(simpleJitLimit) :
-            static_cast<uint16>(simpleJitLimit) - callCount - 1;
+            static_cast<uint16>(simpleJitLimit) - static_cast<uint8>(callCount) - 1;
```

Comment:
The call count can't be > 255 while we're decrementing toward the simple JIT limit. We could probably do without the static cast, but I didn't want any static tools complaining about overflow.